CAAS-poc.md

```diff
@@ -98,7 +98,19 @@ TODO:
 
 Open multi-cluster issues: (1 management : n customer)
 * we have to create restConfig / kubeClients dynamically
+  * base for discovery will be the name of the `BackupStorageLocation`
+  * must be created together with new cluster: 
+    * kubeconfig
+    * `BackupStorageLocation`
+    * s3 credentials (same secret for all but ini key must match cluster ID, just keep entry if cluster gets deleted?)
```

Comment:
Do I assume correctly that this allows us to use separate s3 credentials per cluster?

(But to be honest, when all backups are done by the same process anyway, I don't really see any reason anymore to spend the effort to generate s3 credentials per cluster)

Reply:
As per proposal, we will go for a single credential.